File: articles/structuring-interdisciplinary-knowledge/fortran/interdisciplinary_metrics.f90
program interdisciplinary_metrics
  implicit none

  real :: concepts
  real :: crosswalks
  real :: scope_note_coverage
  real :: method_context_coverage
  real :: traceability
  real :: false_equivalence_risk

  concepts = 8.0
  crosswalks = 6.0
  scope_note_coverage = 7.0 / concepts
  method_context_coverage = 7.0 / concepts
  traceability = 5.0 / crosswalks
  false_equivalence_risk = 1.0 / crosswalks

  print *, "Interdisciplinary Structure Metrics"
  print *, "Scope note coverage: ", scope_note_coverage
  print *, "Method context coverage: ", method_context_coverage
  print *, "Relationship traceability: ", traceability
  print *, "False equivalence risk: ", false_equivalence_risk
end program interdisciplinary_metrics

File: articles/structuring-interdisciplinary-knowledge/cpp/interdisciplinary_metrics.cpp
#include <iostream>
#include <iomanip>

int main() {
    double concepts = 8.0;
    double crosswalks = 6.0;

    std::cout << std::fixed << std::setprecision(3);
    std::cout << "Scope note coverage: " << 7.0 / concepts << "\n";
    std::cout << "Method context coverage: " << 7.0 / concepts << "\n";
    std::cout << "Relationship traceability: " << 5.0 / crosswalks << "\n";
    std::cout << "False equivalence risk: " << 1.0 / crosswalks << "\n";

    return 0;
}
